homeassistant/components/envisalink.py

```diff
@@ -74,9 +74,7 @@
             vol.All(vol.Coerce(int), vol.Range(min=3, max=4)),
         vol.Optional(CONF_EVL_KEEPALIVE, default=DEFAULT_KEEPALIVE):
             vol.All(vol.Coerce(int), vol.Range(min=15)),
-        vol.Optional(CONF_ZONEDUMP_INTERVAL,
-                     default=DEFAULT_ZONEDUMP_INTERVAL):
-            vol.All(vol.Coerce(int), vol.Range(min=15)),
+        vol.Optional(CONF_ZONEDUMP_INTERVAL): vol.Coerce(int),
```

Comment:
Ugh- can't believe i did that. No, I'd rather I actually USE it :).  Another commit is coming soon where i'll put the default back in.  The goal was simply to remove the minimum of 15.